### PR TITLE
feat: LT-1230 - Open Licence Types (Media)

### PR DESCRIPTION
**Pull request by iichr**
- Hide the matrix for goods to countries when finalising

### CODE REVIEW

File: templates/case/advice/final.html [hunk 1]
```diff
@@ -39,9 +39,15 @@ <h1 class="govuk-heading-l">
...
 								{% if case.application.case_type.sub_type.key == 'open' %}
-									<a class="govuk-button"  href="{% url 'cases:finalise_goods_countries' queue.id case.id %}" id="finalise_button">
-										{% lcs 'cases.ViewAdvicePage.Actions.FINALISED_GOODS_AND_COUNTRIES' %}
-									</a>
+									{% if case.application.goodstype_category.key == 'media' %}
+										<a class="govuk-button"  href="{% url 'cases:finalise' queue.id case.id %}" id="finalise_button">
+										{% lcs 'cases.ViewAdvicePage.Actions.FINALISE' %}
+										</a>
+									{% else %}
+										<a class="govuk-button"  href="{% url 'cases:finalise_goods_countries' queue.id case.id %}" id="finalise_button">
```
Comment (jfaracik) [inline templates/case/advice/final.html:47]: Use a {% govuk_link_button %}

Comment (iichr) [reply templates/case/advice/final.html:47]: Copied that from just underneath, should we change both then?

Comment (iichr) [reply templates/case/advice/final.html]: Have a look changed it in a few places :)

File: cases/views/advice.py [hunk 2]
```diff
@@ -326,7 +326,7 @@ def get(self, request, *args, **kwargs):
         case = get_case(request, str(kwargs["pk"]))
         case_type = case["application"]["case_type"]["sub_type"]["key"]
 
-        if case_type == CaseType.OPEN.value:
+        if case_type == CaseType.OPEN.value and (not case["application"]["goodstype_category"]["key"] == "media"):
```
Comment (a-bek) [inline cases/views/advice.py:329]: can you use `!=` instead of `not`?

Comment (iichr) [reply cases/views/advice.py:329]: On it

File: templates/case/advice/final.html [hunk 3]
```diff
@@ -39,13 +39,16 @@ <h1 class="govuk-heading-l">
 							</button>
 							{% if can_advice_be_finalised %}
 								{% if case.application.case_type.sub_type.key == 'open' %}
-									<a class="govuk-button"  href="{% url 'cases:finalise_goods_countries' queue.id case.id %}" id="finalise_button">
-										{% lcs 'cases.ViewAdvicePage.Actions.FINALISED_GOODS_AND_COUNTRIES' %}
-									</a>
+									{% if case.application.goodstype_category.key == 'media' %}
+										{% make_list queue.id case.id as button_params %}
```
Comment (jfaracik) [inline templates/case/advice/final.html:43]: Could you just move the `{% make_list queue.id case.id as button_params %}` out of the if statement if its being reused?

Also pedantic, but IDs use dashes, not underscores :)

Comment (iichr) [reply templates/case/advice/final.html]: Done, changed to dashes as well

**Pull request merged**